lib/Excel/Writer/XLSX.pm

```diff
@@ -448,15 +448,19 @@ The properties that can be set are:
     title
     subject
     author
+    last_author
     manager
     company
     category
```

Comment:
The W was an attempt to either zoom or scroll on my phone. I'll delete the comment(s).